core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/BlockContext.scala

```diff
@@ -74,6 +75,13 @@ object BlockContext {
   object Outcome {
     final case class Exist[T](value: T) extends Outcome[T]
```

Comment:
Isn't it `Option`?

Reply:
semantics is the same, but IMO type alias is not enough. now, IMO, is readable and descriptive